package.json

```diff
@@ -78,5 +79,8 @@
   "bugs": {
     "url": "https://github.com/opynfinance/GammaProtocol/issues"
   },
-  "homepage": "https://github.com/opynfinance/GammaProtocol#readme"
+  "homepage": "https://github.com/opynfinance/GammaProtocol#readme",
+  "dependencies": {
+    "@openzeppelin/contracts": "^3.1.0"
```

Comment:
we shouldn't' need this package

Reply:
so we should use this one https://github.com/OpenZeppelin/openzeppelin-sdk/blob/master/packages/lib/contracts/ownership/Ownable.sol ?

Reply:
why not ?